backend/apps/matching/models/staff.py

```diff
@@ -7,8 +7,6 @@ class Staff(models.Model):
     """An admin class for organizing the different access classes for maintainers of the platform."""
```

Comment:
Is Staff class necessary at all anymore? I don't see what it could be used for. 
I'd suggest removing the Model

Reply:
See #22, not scope of this issue